### ISSUE TITLE
Confusing account activation Emails

### ISSUE THREAD

**Issue by paulstroot** [ocupop/comission-theme#17]
Created an account and got email asking me to verify address, however at same time I also received a second confirmation email saying I’d activated it when I hadn’t. However the process is technically OK as it still wouldn’t let me access the account until I had activated it. Although I didn’t get another confirmation email once I did actually activate it. Just need second email to go out at correct time.

**Comment by paulstroot** [comment]
I was able to recreate this. Here are the issues that I see

- When submitting the "Create Account" form, but the "account activation" and "account confirmation" emails get sent out immediately, even though the account has not yet been activated.
- When trying to log in without activating the account, a new account activation email is sent. I'm not sure if this is by design or not.

**Comment by paulstroot** [comment]
Tom and I did some investigating here, and it turns out that I think we are misinterpreting the intention of the customer welcome email. This is sent out as soon as the user submits the form by design, and the activation email is also sent out, so it is working as intended. What we need to do is to update the wording on the welcome email. Below is a screenshot of what I have changed it to for now, If you would like to fine-tune, you can find the copy under settings -> notifications -> Customer Notifications -> Customer Account Welcome
![image](https://github.com/user-attachments/assets/644451c6-8c47-4133-a5cc-f51716481033)

**Comment by ZiggyAtl** [comment]
@paulstroot thanks for that. I just want to make sure I've not miscommunicated what happened. The first email was one saying I've created an account and all I need to do is activate it, with a link to click to activate. See below
![Screen Shot 2024-11-21 at 9 16 19 PM](https://github.com/user-attachments/assets/0e01e171-ff4c-4810-9b18-51ac0659df75)

The other email came seconds after that (and without me having clicked the activate) and said that I have now activated my account, and giving a link to the store to shop. See below. I clicked on that link and it wouldn't let me in the store and said I needed to activate my account first. So I went back to the first email and clicked the link.
![Screen Shot 2024-11-21 at 9 17 53 PM](https://github.com/user-attachments/assets/f7cd6083-1ee3-4a3f-bca5-5ee90354e717)

I can't see how that second email is working as intended because the description of it in Shopify is 'sent when a customer completes their account activation' but it's sending when the account is created. If it's easier to fix that and have it go out once the activation link has been clicked in the first email I'd rather do that. But if it can't easily be fixed I think the solution you proposed is the best and I'm happy to go with that version you prepared.

**Comment by ZiggyAtl** [comment]
@paulstroot as an add on to the previous comment we've been testing again tonight and this time it allowed the new user to access their account without going through the activate process. Has someone changed that at your end today? For security purposes I would like to maintain the activate process.

**Comment by paulstroot** [comment]
I see exactly what you're saying, and the fact that the Shopify admin states "Sent when a customer completes their account activation" for the "Customer Account Welcome" email absolute does make it sound like it shouldn't be sent until the customer has activated their account.

This is an issue with Shopify, because it is certainly not the behavior that we are experiencing, and should be worked through with the [Shopify Help Center](https://help.shopify.com/en).  Reach out to them and let them know what we're experiencing and they should be able to help resolve this issue for you.

**Comment by paulstroot** [comment]
Yes. Let's not change the updated wording. I will not make any updates to it. You can feel free to make updates if you'd like to fine-tune the wording at all.
